package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "boilermaker",
```

Comment:
Looks like name change got overridden with pulling the finished version of boilermaker